### ISSUE TITLE
deepspeed "--fp16" parameter dynamic loss scaling issue

### ISSUE THREAD

**Issue by afiaka87** [lucidrains/DALLE-pytorch#186]
When training with fp16 - deepspeed attempts to find a good value for loss scaling right away - but it starts entirely too high and has always dropped to 2^15 for me. As such - it's a good idea to tell deepspeed to go ahead and start there from iteration 0. 

```
    'fp16': {
        'enabled': args.fp16,
        'loss_scale': 0,
        'initial_scale_power': 15,
    },
```

**Comment by robvanvolt** [comment]
Does it prevent the infinite loss for large batch sizes? =D

**Comment by afiaka87** [comment]
> Does it prevent the infinite loss for large batch sizes / did you already run experiments or is it a proposal?

Sorry - don't think so. I believe that solution requires further digging.
